alerts.md

```diff
@@ -193,46 +194,78 @@ Example Helm values.yaml:
 
 Enabling diagnostic alerts in Kubecost occur when an event impacts product uptime. This feature can be enabled in seconds from a values file. The following health events are detected:
 
-* Prometheus is unreachable
-* Kubecost metrics missing over last 5 minutes
-* More coming soon.
+- Prometheus is unreachable
+- Kubecost metrics missing over last 5 minutes
+- More coming soon.
 
 This alert only uses Slack (email coming soon), so it requires the `globalSlackWebhookUrl` field.
 
 Example Helm values.yaml:
+
 ```
         # Kubecost Health Diagnostic
         - type: diagnostic
-          window: 10m               
- ```
-
-*Versions Earlier than 1.79.0*
+          window: 10m
+```
 
-This alert used to be configured via the `notifications.alertConfigs.kubecostHealth` flag seen [here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/31dc60d2c539720f2b2a72c8e22b2f6b866580bd/cost-analyzer/values.yaml#L31). If upgrading to version 1.79.0 or newer, remove the `kubecostHealth` flag, and append the alert definition shown above. 
+_Versions Earlier than 1.79.0_
 
+This alert used to be configured via the `notifications.alertConfigs.kubecostHealth` flag seen [here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/31dc60d2c539720f2b2a72c8e22b2f6b866580bd/cost-analyzer/values.yaml#L31). If upgrading to version 1.79.0 or newer, remove the `kubecostHealth` flag, and append the alert definition shown above.
 
 ### Type: Cluster Health
 
 Cluster health alerts occur when the cluster health score changes by a specific threshold. The health score is calculated based on the following criteria:
 
-* Low Cluster Memory
-* Low Cluster CPU
-* Too Many Pods
-* Crash Looping Pods
-* Out of Memory Pods
-* Failed Jobs
+- Low Cluster Memory
+- Low Cluster CPU
+- Too Many Pods
+- Crash Looping Pods
+- Out of Memory Pods
+- Failed Jobs
 
-This alert only uses Slack (email coming soon), so it requires the `globalSlackWebhookUrl` field. 
+This alert only uses Slack (email coming soon), so it requires the `globalSlackWebhookUrl` field.
 
 Example Helm values.yaml:
 
 ```
-        # Health Score Alert 
+        # Health Score Alert
         - type: health              # Alerts when health score changes by a threshold
           window: 10m
           threshold: 5              # Send Alert if health scores changes by 5 or more
 ```
 
+## Configuring Alerts in the Kubecost UI
+
+### Cluster Health and Diagnostic Alerts
+
+Cluster Health Alerts and Diagnostic Alerts work differently from other alert types. While other alerts monitor cost data for cost or efficiency anomalies, Health and Diagnostics montior the health of Kubecost itself, as well as the health of the cluster running Kubecost. The UI treats these alert types as "on" or "off", managing a single instance of each, and allowing the settings of this single instance to be adjusted.
+
+![Kubecost Health Alerts](images/health_and_diagnostics_alert_controls.png)
```

Comment:
Style guide also specifies something a little odd for image links, presumably so they continue to work after the doc is converted for zendesk: https://github.com/kubecost/docs/blob/main/markdown.md#images

Reply:
Much appreciated. Will make the requisite adjustments to image links and my Prettier config. Thanks Michael!